chemdash.py

```diff
@@ -367,6 +374,7 @@ def get_dist_plot(selected_columns, data):
 
 
 if __name__ == '__main__':
-
-        app.run_server(debug=True)
-
+    port = 8000
+    print(f"Launching server: http://0.0.0.0:{port}")
+    app.run_server(host="0.0.0.0", port=port, debug=True)
+    print(f"Server exited.")
```

Comment:
The only other update to the python code just to make it visible in the terminal when it is time to open the browser.